sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -195,14 +202,31 @@ export class ServiceBusSenderImpl implements ServiceBusSender {
           throw error;
         }
       }
-
-      return this._sender.sendBatch(batch, options);
     } else if (isServiceBusMessageBatch(messages)) {
-      return this._sender.sendBatch(messages, options);
-    } else if (isServiceBusMessage(messages)) {
-      return this._sender.send(messages, options);
```

Comment:
Note for future clean up:

Now that we dont use the single send method from the `MessageSender` class anymore, it can be deleted. Same for `sendMessages()` method in the `MessageSender` class

Reply:
Sounds good.

Reply:
Logged https://github.com/Azure/azure-sdk-for-js/issues/11687